CYF-London | Amir Montazeri | Model User Focused Data | Week 1

Amirmontazerii opened the pull request:
<!--

You must title your PR like this:

COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK

For example,

NW4 | Carol Owen | HTML-CSS-Module | Week1

Complete the task list below this message.
If your PR is rejected, check the task list.

-->

## Learners, PR Template

Self checklist

- [x] I have committed my files one by one, on purpose, and for a reason
- [x] I have titled my PR with COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK 
- [x] I have tested my changes
- [x] My changes follow the [style guide](https://syllabus.codeyourfuture.io/guides/code-style-guide/)
- [x] My changes meet the [requirements](./README.md) of this task

## Changelist

I changed a part of paragrapgh in header in Cakes-Co index.html from best cake to most delicious cake

## Questions

Ask any questions you have for your reviewer.

Amirmontazerii committed:
change the best to most delicious in header

Amirmontazerii committed:
fixing the html tag

Amirmontazerii committed:
Making a website about Github

SallyMcGrath commented on Cakes-Co/index.html:
```diff
@@ -25,7 +25,7 @@
           </nav>
         </aside>
-        <p class="motto-p">The best cakes in town delivered to your door</p>
+        <p class="motto-p">The most delicious cakes in town delivered to your door</p>
```
I don't think this change belongs in this PR. Only changes inside the wireframe folder belong in this PR.

Amirmontazerii replied:
Sorry, this was my bad

SallyMcGrath commented on Wireframe/index.html:
```diff
@@ -3,11 +3,40 @@
+      </header>
+
+      <main>
+        <section>
```
What is this section doing? Can you explain what section "means"  in HTML?

Amirmontazerii replied:
basically The <section> element in HTML is used to define a thematic grouping of content, and I think I didn't know what is it exactly and I tried to google it and after some research I saw and read this : https://stackoverflow.com/questions/7549561/difference-between-section-vs-article-elements-in-html
and I got conclusion that I used a main <section> that have three article in it.

SallyMcGrath reviewed (changes requested):
Thanks for this @Amirmontazerii . It's a good start, but I think there's quite a bit more to do.

It doesn't look very much like the wireframe. Do you need help understanding how to "lay out" your web page to match the picture? Come to Slack and pair with someone on this if that's helpful.

Amirmontazerii committed:
add some css and a comment in html

Amirmontazerii committed:
change in button and fixed the wrong folder

Amirmontazerii committed:
deleted wrong folder

cjyuan commented:
- May I suggest using a HTML validator such as https://validator.w3.org/ to check for potential error in HTML code?

- Your page does not look quite like the wireframe yet. Typical things to pay attention to in a wireframe include:
   - Text alignments and justification
   - Relative size and position of every elements.
 
 - The footer is not **fixed** at the bottom of the ***viewport***; it should stay at the bottom of the viewport even when the page content is scrolled. 
 
 - I could not find any use of the `:first-child` pseudo class selector (as required by one of the acceptance criteria)

Amirmontazerii committed:
layout fixed

Amirmontazerii committed:
button method changed from a to onclick

Amirmontazerii commented:
@cjyuan Thank you for your review, it was so helpful, I changed the layout and also footer is fixed now. I used the validator as well for my HTML codes and changed button method from a inside to onclick.
